js/src/lib/Details/index.js

```diff
@@ -54,18 +54,12 @@ class Details extends React.Component {
       this.state.githubRepo.user &&
       this.state.githubRepo.project
     ) {
-      get({
-        url: this.state.changelogFileName
-          ? this.state.changelogFileName
-          : prefixURL('CHANGELOG.md', {
-              base: 'https://raw.githubusercontent.com',
-              user: this.state.githubRepo.user,
-              project: this.state.githubRepo.project,
-              head: this.state.gitHead ? this.state.gitHead : 'master',
-              path: this.state.githubRepo.path.replace(/\/tree\//, ''),
-            }),
-        type: 'text',
-      }).then(res => this.setState({ changelog: res }));
+      if (this.state.changelogFilename) {
+        get({
```

Comment:
this is actually kind of separate, but it could've been changed earlier. Basically at indexing time (our backend), we check a lot of possible file names, and CHANGELOG.md is always tested as well, so a fallback isn't needed in this case.